vsintegration/src/FSharp.Editor/LanguageService/SymbolHelpers.fs

```diff
@@ -162,70 +160,3 @@ module internal SymbolHelpers =
             let usesByDocumentId = symbolUsesWithDocumentId |> Seq.groupBy fst
             return usesByDocumentId.ToImmutableDictionary(fst, snd >> Seq.map snd >> Seq.toArray)
         }
-
-    type OriginalText = string
-
-    // Note, this function is broken and shouldn't be used because the source text ranges to replace are applied sequentially,
```

Comment:
This is indeed a very buggy something with a dubious purpose. Removing for good.